scripts/newsletter.js

```diff
@@ -83,6 +111,30 @@ async function bountyWinner (q) {
   }
 }
 
+async function getTopUsers ({ by, cowboys = false, includeHidden = false, count = 5, when = 'week' } = {}) {
+  const accum = []
+  let cursor = ''
+  while (accum.length < count) {
+    let variables = {
+      cursor
+    }
+    if (!cowboys) {
+      variables = {
+        ...variables,
+        by,
+        when
+      }
+    }
+    const result = await client.query({
+      query: cowboys ? TOP_COWBOYS : TOP_USERS,
+      variables
+    })
+    cursor = result.data[cowboys ? 'topCowboys' : 'topUsers'].cursor
+    accum.push(...result.data[cowboys ? 'topCowboys' : 'topUsers'].users.filter(user => includeHidden ? true : !!user))
+  }
+  return accum.slice(0, count)
+}
```

Comment:
This logic is more complicated than originally intended, since the request was to not include hidden stackers. That means, in theory, fetching the first page of 21 records could not give us 5 non-hidden stackers, if many of the top users are hidden. So this code will request page after page until we have 5 non-hidden stackers. In practice, this is unlikely to be an issue and likely only 1 page of data is required, but I wanted to make the code bullet-proof :)